index.html

```diff
@@ -75,7 +75,7 @@ <h1 class="title">Hello There!<br /> Welcome to my Website!</h1>
   </section>
   <main>
     <section id="portfolio" class="my-works">
-      <div class="work-item">
+      <div class="work-item" onclick="openDetails(1)">
         <img alt="work-item-img" src="./assets/img/work/work-1-cover.png">
         <div class="work-item-content">
           <h3 class="work-title">Tonic</h3>

```

Comment:
https://github.com/joshuaivie/Personal-Portfolio/blob/0b72aee705790b73a6cff7fc74523359a1bcb6b2/index.html#L78-L163

- [ ] As required :point_down: 

![image](https://user-images.githubusercontent.com/57421823/151015393-2368bdbc-0f28-46f4-9357-b17c3811e52d.png)

As seen above, all the projects section is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). This requires changes in existing code - already created HTML code for projects list should be removed and instead it should be added via JavaScript.

This means you can use the `projects` array and logic that you already have, to populate the projects section of the main page entirely with javascript!

If you are struggling or don't know how to do so, here is an [example](https://github.com/olawale-o/portfolio-mobile-version/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95)

 - All those cards information should be stored into an array of objects then create a function to render through the array and create dynamically your HTML with the information from the object array, the works section must stay empty